cli/command/system/prune.go

```diff
@@ -32,6 +34,7 @@ func NewPruneCommand(dockerCli *command.DockerCli) *cobra.Command {
 	flags := cmd.Flags()
 	flags.BoolVarP(&opts.force, "force", "f", false, "Do not prompt for confirmation")
 	flags.BoolVarP(&opts.all, "all", "a", false, "Remove all unused images not just dangling ones")
+	flags.Var(&opts.filter, "filter", "Provide filter values (i.e. 'until=<timestamp>')")
```

Comment:
Should probable be `e.g.` instead of `i.e.` since there's going to be other filters included eventually (would prevent forgetting changing the text here)

The same applies for the other prune commands